### PR TITLE
Adding Normalize by Color Luminance for Power unit.

### PR DESCRIPTION
**Pull request by michelchafouin**
Set to False by default to mimic previous behavior and Cycles. Power is a radiometric unit and should not be normalized.

WARNING: Point, Laser, Area and Emissive materials work fine, but Spotlights export with normalizeycolor = True no matter what is set in the UI. Can't find the cause, please help me with that.

### CODE REVIEW

File: properties/light.py [hunk 1]
```diff
@@ -24,6 +24,11 @@
     "this feature and uses only the light gain"
 )
 
+NORMALIZEBYCOLOR_DESCRIPTION = (
+    "Normalize intensity by the Color Luminance; Radiometric units like Power should not be "
+    "normalized, while Photometric units like Lumen, Candela, Lux should."
```
Comment (Theverat) [inline properties/light.py]: Descriptions should not end with a dot, Blender adds it afterwards (and warns if there is already a dot).

**Comment by Theverat** [comment]
I can't reproduce the problem with the spotlights, if I disable the checkbox the properties are created like I would expect:
```
light props:
scene.lights.1737221504312.gain = 1 1 1
scene.lights.1737221504312.importance = 1
scene.lights.1737221504312.id = 0
scene.lights.1737221504312.type = "spot"
scene.lights.1737221504312.coneangle = 22.50000062611954
scene.lights.1737221504312.conedeltaangle = 3.375000228028386
scene.lights.1737221504312.color = 1 1 1
scene.lights.1737221504312.efficency = 17
scene.lights.1737221504312.power = 100
scene.lights.1737221504312.normalizebycolor = 0
scene.lights.1737221504312.position = 0 0 0
scene.lights.1737221504312.target = 0 0 -1
```
Can you post a testscene where it's not working?

**Comment by michelchafouin** [comment]
http://pasteall.org/blend/index.php?id=52787
The Spotlight is always normalized for me.

**Comment by Theverat** [comment]
The property is being set correctly though.
It might be a problem in LuxCore during property parsing?

**Comment by Theverat** [comment]
Looks to me like it's missing here: https://github.com/LuxCoreRender/LuxCore/blob/72f5f7441648a38d04166a984e0139068df0cb43/src/slg/scene/parselights.cpp#L360

**Comment by Dade916** [comment]
I should have fixed the problem.

**Comment by Theverat** [comment]
Thanks!

**Pull request merged**